server.js

```diff
@@ -69,70 +89,59 @@ var newBookUUID = 18;
 
 // define a root route: localhost:3000/
 app.get('/', function (req, res) {
-  res.sendFile('views/index.html' , { root : __dirname});
+	res.sendFile('views/index.html' , { root : __dirname});
 });
 
 // get all books
 app.get('/api/books', function (req, res) {
-  // send all books as JSON response
-  console.log('books index');
-  res.json(books);
+	// send all books as JSON response
+	console.log('books index');
+	DB.Book.find({}, (err, allBooks) => {
+		if (err) throw err;
+		res.json({users: allBooks})
```

Comment:
watch your variable and argument names. "users" here doesn't make sense for the key name of books.